components/CopyAddress.tsx

```diff
@@ -1,10 +1,15 @@
 import copy from "copy-to-clipboard";
 import CopyIcon from "./icons/CopyIcon";
+import useTheme from "../hooks/useTheme";
 
 const CopyAddress = ({ address, url }: { address: string; url?: string }) => {
+  const isDarkMode = useTheme();
+
   return (
     <div
-      className="-ml-1 inline-flex cursor-pointer items-center px-1 font-mono hover:bg-dark hover:text-white active:bg-darkGray3"
+      className={`-ml-1 inline-flex cursor-pointer items-center px-1 font-mono ${
+        isDarkMode ? "hover-dark" : "hover-light"
+      }`}
```

Comment:
You should be able to use the `dark` typescript selector e.g. `hover-dark dark:hover-light`

Reply:
this is more straightforward and cleaner, i totally forgot about this option